src/users/views.py

```
@@ -37,3 +39,14 @@ def get_user_data(self, instance):
             return Response(UserSerializer(self.request.user, context={'request': self.request}).data, status=status.HTTP_200_OK)
         except Exception as e:
             return Response({'error': 'Wrong auth token' + e}, status=status.HTTP_400_BAD_REQUEST)
+
+    @action(detail=False, methods=['get'], url_path='verify', url_name='verify_account')
+    def verify_account(self, request):
+        token = request.query_params.get('token') or ''
+        valid, user = default_token_generator.check_token(token)
```

Maybe throw exception from token generator if it's invalid